src/qiskit_iqm/test.txt

```diff
@@ -0,0 +1,30 @@
+from qiskit import QuantumCircuit
```

Comment:
Was this file added in error? It doesn't seem like an actual test file.

Reply:
yes, thanks! Removed